letsencrypt/client/domain.sls

```diff
@@ -16,6 +23,9 @@ certbot_{{ domain }}:
         --{{ auth.method }}
         {%- endif %}
         -d {{ params.name|default(domain) }}
+        {%- for d in params.get('multi', []) %}
```

Comment:
I would rename `multi` to `names` as it's convention to have plural naming for lists and `names` seems to be more natural as above is `name`. So you can have:
```
domain.org:
  names:
    - www.domain.org
```

or:
```
my_domain:
  name: domain.org
  names:
    - www.domain.org
```